src/lib/stitching/kaws/stitching.ts

```diff
@@ -117,6 +117,10 @@ export const kawsStitchingEnvironment = (
         resolve: (parent, _args, context, info) => {
           const query = parent.query
           const hasKeyword = Boolean(parent.query.keyword)
+          const contextWithCache = { ...context }
+          const filterArtworksLoader = context.filterArtworksLoaderWithCache
+          contextWithCache.filterArtworksLoader = filterArtworksLoader
```

Comment:
Did y’all try something like the following instead of defining a new data loader (that has a cache setting specific to this use-case)?

```js
const contextWithCache = {
  ...context,
  filterArtworksLoader: () => context.filterArtworksLoader({}, { requestThrottleMs: 1000 * 60 * 60 })
}
```